build.gradle

```diff
@@ -112,16 +114,21 @@ def afuChild      = fromBaseDir.curry(afuDir)
 def plumeDir        = env["PLUME"] ?: jsr308Child("plume-lib")
 def plumeChild      = fromBaseDir.curry(plumeDir)
 
+def langtoolsChild = fromBaseDir.curry(jsr308_langtools)
+
 //JarsToPackage contains both all members that should be on the classpath for this build
 //and all jars (except for checker-framework-inference.jar) that should be included in
 //the jar produced by the allJars command
 def jarsToPackage = [ //Annotation File Utilities paths        //TODO: AFU Jar has some things that are also in checkers.jar
                         "annotation-file-utilities.jar",
                     ].collect { afuChild(it).getAbsolutePath()      } +
+                    [
+                        "dist/lib/javac.jar",
+                    ].collect { langtoolsChild(it).getAbsolutePath() } +
 
                     [//checker relative paths
-                        "checker/dist/$jreJarName",
-                        "checker/dist/javac.jar",
+                        //"checker/dist/$jreJarName",
```

Comment:
> A comment why this is commented out would be good.

Sure, will do it.

> Currently none of the checkers uses the annotated JDK, so we don't require it. However, this will result in a warning when running a checker, right?

Yes, if we running a checker with normal javac like `javac -cp checker/dist/checker.jar -processor aChecker aFile.java`, it will issue an warning if we don't have annotated jdk.

If we running a checker by checker framework javac, then it will throw an Exception of doesn't find the annotated jdk.